.gitignore

```diff
@@ -21,3 +21,4 @@ coverage
 
 # Ignore application configuration
 /config/application.yml
+/public
```

Comment:
Hey, Ben, our updates to the 404 and other pages are in this file, so I'm not sure they should be ignored.  Is there something else happening in these files that I'm missing?

Reply:
This is convenient for not adding our precompiled assets and getting the project running in production

Reply:
I made the change we talked about. public/assets should be fine to ignore